sensing/autoware_pointcloud_preprocessor/src/filter.cpp

```diff
@@ -123,8 +125,8 @@ void autoware::pointcloud_preprocessor::Filter::setupTF()
     }
     has_static_tf_only_ = true;
   }
-  managed_tf_buffer_ =
-    std::make_unique<autoware_utils::ManagedTransformBuffer>(this, has_static_tf_only_);
+  managed_tf_buffer_ = std::make_unique<autoware_utils::ManagedTransformBuffer>(
+    this, has_static_tf_only_, use_pc_stamp_for_dynamic_transform_lookup_);
```

Comment:
Same comment with taekjin-san.
Do we need transform within current timestamp for pointcloud transforms? If no, we can simply set true in this part.